docs/tempo/website/getting-started/quickstart-tempo.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Quickstart with Tempo
+---
+
+If you are running an application locally that is already instrumented with tracing,
+this guide will help set up the components required to send & query traces in Tempo.
+
+## Step 1: Spin up Tempo backend
+
+First, set up a docker network as shown -
+
+```
+docker network create docker-tempo
+```
+
+Next, download the [configuration file](https://github.com/grafana/tempo/blob/master/example/docker-compose/etc/tempo-local.yaml) using the following command -
+
+```
+curl -o tempo-local.yaml https://raw.githubusercontent.com/grafana/tempo/master/example/docker-compose/etc/tempo-local.yaml
+```
+
+The config file above configures Tempo to listen on default ports for a number of protocols.
+List of protocols and their default ports:
+
+|  Protocol    |   Port  |
+|  ---         |   ---   |
+|  OpenTelemetry  | 55680 |  # Grafana Agent uses this.
```

Comment:
This is the legacy gRPC port. The new one is 4317. See https://github.com/open-telemetry/opentelemetry-collector/blob/08a18d5b9f1b8ee001f8e16e6f6b9daf45b4d42e/receiver/otlpreceiver/factory.go#L45

Maybe it's something interesting to change (on the Agent, I suppose).

Reply:
Good catch, updated.

Reply:
Wonder if we need to update our agent ports as well.

Reply:
I saw that the port is used on some agent examples/configs. Maybe it's a good idea to change them!

Here, in our repo, we use the legacy port on some other places: https://github.com/grafana/tempo/search?q=55680